source/reference/minio-mc/mc-stat.rst

```diff
@@ -154,36 +139,119 @@ Parameters
+   :class: copyable
 
-         mc stat ALIAS/PATH
+   mc stat myminio/mybucket/myfile.txt
 
-      - Replace :mc-cmd:`ALIAS <mc stat ALIAS>` with the 
-        :mc:`alias <mc alias>` of the S3-compatible host.
+The output resembles the following:
 
-      - Replace :mc-cmd:`PATH <mc stat ALIAS>` with the path to the bucket
-        or object on the S3-compatible host.
+.. code-block:: shell
 
-   .. tab-item:: Object(s) in Bucket
+   Name      : myfile.txt
+   Date      : 2024-07-16 15:40:02 MDT 
+   Size      : 6.0 KiB 
+   ETag      : 3b38f7b05a0c42acdc377e60b2a74ddf 
+   Type      : file 
+   Metadata  :
+     Content-Type: text/plain 
 
-      Use :mc:`mc stat` with the :mc-cmd:`~mc stat --recursive` option
-      to apply the operation to all objects in the bucket:
+You can also specify more than one object:
 
-      .. code-block:: shell
-         :class: copyable
+.. code-block:: shell
+   :class: copyable
+
+   mc stat myminio/mybucket/file1.txt play/yourbucket/file2.txt
+
+To display detail for all objects in a bucket, use :mc-cmd:`~mc stat --recursive`:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc stat --recursive myminio/mybucket
+
+The output resembles the following:
+
+.. code-block:: shell
+
+   Name      : file1.txt
+   Date      : 2024-07-16 15:40:02 MDT
+   Size      : 6.0 KiB
+   ETag      : 3b38f7b05a0c42acdc377e60b2a74ddf
+   Type      : file
+   Metadata  :
+     Content-Type: text/plain
+
+   Name      : file2.txt
+   Date      : 2024-07-26 10:45:19 MDT
+   Size      : 6.0 KiB
+   ETag      : 3b38f7b05a0c42acdc377e60b2a74ddf
+   Type      : file
+   Metadata  :
+     Content-Type: text/plain
+
+
+Display Bucket Details
+~~~~~~~~~~~~~~~~~~~~~~
+
+The following example displays information about the bucket ``mybucket`` on the ``myminio`` MinIO deployment:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc stat myminio/mybucket
+
+The output resembles the following:
+
+.. code-block:: shell
+
+   Name      : mybucket
+   Date      : 2024-07-26 10:56:43 MDT 
+   Size      : N/A    
+   Type      : folder 
+
+   Properties:
+     Versioning: Un-versioned
+     Location: us-east-1
+     Anonymous: Disabled
+     ILM: Disabled
+
+   Usage:
+         Total size: 6.0 KiB
+      Objects count: 1
+     Versions count: 0
+
+   Object sizes histogram:
+      1 object(s) BETWEEN_1024B_AND_1_MB
+      1 object(s) BETWEEN_1024_B_AND_64_KB
+      0 object(s) BETWEEN_10_MB_AND_64_MB
+      0 object(s) BETWEEN_128_MB_AND_512_MB
+      0 object(s) BETWEEN_1_MB_AND_10_MB
+      0 object(s) BETWEEN_256_KB_AND_512_KB
+      0 object(s) BETWEEN_512_KB_AND_1_MB
+      0 object(s) BETWEEN_64_KB_AND_256_KB
+      0 object(s) BETWEEN_64_MB_AND_128_MB
+      0 object(s) GREATER_THAN_512_MB
+      0 object(s) LESS_THAN_1024_B
+
+
+Count of Objects in a Bucket
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To show the number of objects in a bucket, use :mc:`mc stat --json` and extract the value of ``objectsCount`` with a JSON parser:
```

Comment:
I swear I got refs including global flags to link correctly at some point. I forget how.